hearpreprocess/pipeline.py

```diff
@@ -747,18 +827,7 @@ def run(self):
         # But we aren't going to use audio that is more than a couple
         # minutes or the timestamp embeddings will explode
         sample_duration = self.task_config["sample_duration"]
-
-        # Get the max split duration from the task config.
-        # If not defined use defaults from MAX_TASK_DURATION_BY_SPLIT
-        if "max_task_duration_by_split" in self.task_config:
-            max_task_duration_by_split = self.task_config["max_task_duration_by_split"]
-            # Check if all the splits are specified in
-            # max_task_duration_by_split
-            assert set(max_task_duration_by_split.keys()) == set(SPLITS)
-            max_split_duration = max_task_duration_by_split[self.split]
-        else:
-            # Get the default values for the split
-            max_split_duration = MAX_TASK_DURATION_BY_SPLIT[self.split]
+        max_split_duration = self.get_max_split_duration()
```

Comment:
see like this could have been a dict lookup but now is a function, seems like a nice way to wrap higher-level config stuff.